pkg/config/config.go

```diff
@@ -19,20 +20,20 @@ type GuidPoolConfig struct {
 }
 
 func (dc *DaemonConfig) ReadConfig() error {
-	glog.Info("ReadConfig():")
+	log.Info().Msg("ReadConfig():")
```

Comment:
Lets not have function names in  non Debug messages.

Reply:
Yes move it to debug